runtime/themes/gruvbox.toml

```diff
@@ -27,6 +27,7 @@
 "escape" = { fg = "fg2", modifiers = ["bold"] }
 "label" = "aqua1"
 "module" = "yellow1"
+# "variant" TODO
```

Comment:
Drop these, we should only add TODOs if it's a `ui.` change. Otherwise we'll be TODOing every single possible scope combination

Reply:
On second thought, we shouldn't be introducing another toplevel attribute at all. This should be `type` or `type.enum.variant` maybe.